### ISSUE TITLE
UI: show file offsets and virtual address in verbose output

### ISSUE THREAD

**Issue by mr-tz** [mandiant/flare-floss#937]
Display raw and virtual offsets in verbose mode.

This could help to match things up more quickly between FLOSS output, the file, and disassembler view.

**Comment by Om-Thorat** [comment]
Hey, I would like to work on this. Though can you specify more? as to what out is needed? Should there be more columns or should i just log them as info?

**Comment by mr-tz** [comment]
Hi! I imagined extra columns for this, correct.

**Comment by Om-Thorat** [comment]
![Screenshot from 2024-02-27 18-59-44](https://github.com/mandiant/flare-floss/assets/76207818/4a3d6760-2552-4cc5-a355-a81989a2d665)


Will something like this be ideal?

**Comment by mr-tz** [comment]
Yes, however, this doesn't apply so much to decoded strings vs. the language strings (Go and Rust).
If we had something for the language strings that would be amazing!

**Comment by mr-tz** [comment]
Would also be interesting to show (and order by) `referenced from va` or `referenced from fva`.

**Comment by Chee-Tzu** [comment]
Hello, is this still valid?

**Comment by mr-tz** [comment]
Yes ☺️

**Comment by divyansh360** [comment]
Hi @mr-tz, I’m applying for GSoC 2025 and would be happy to make my small contributions here, as I feel this could be a good start for the project relating to language-specific strings. Can I take it?

**Comment by mr-tz** [comment]
Absolutely, thanks.

**Comment by Vineet817** [comment]
Hi @mr-tz, I’m applying for GSoC 2026 and I am looking for small contributions here. 

I noticed that this issue appears to have been addressed by PR #1105, which adds offset and virtual address support for language-specific strings in verbose mode.

However, since the issue is still open, I wanted to ask whether there is any remaining work or follow-up expected for this task.

I would be happy to look into contributing.
